travis/stage-python3.sh

```diff
@@ -1,7 +1 @@
 #!/bin/bash
```

Comment:
@smurching I'll remove this file in a follow-up otherwise I need to fix `.travis.yml`